### PR TITLE
Update onChange of Checkbox to remove isRequired

### LINKED ISSUE #948: Consider removing `isRequired` from the `onChange` propType of `Checkbox`
**Describe the bug**
`onChange` should be supported but it doesn't need to be required on every instance of `Checkbox`

**Affected project(s) (e.g. `pcln-design-system`, `pcln-modal`, etc.)**
core/pcln-design-system

As reported by @BeniCheni

### PR DESCRIPTION
**Pull request by BeniCheni**
Fix #948.

### CODE REVIEW

**Review by pcln-james — APPROVED** [review]
LGTM

**Review by craigpalermo — CHANGES_REQUESTED** [review]
LGTM, but could you run `rush change`?

**Comment by BeniCheni** [comment]
> LGTM, but could you run rush change?

@craigpalermo, pushed the update from `rush change`. PTAL?

File: common/config/rush/command-line.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
       "summary": "Build the Storybook",
       "description": "Build the Storybook",
       "safeForSimultaneousRushProcesses": false,
-      "shellCommand": "(cd apps/storybook && rushx storybook:baseline)"
+      "shellCommand": "(cd apps/storybook && rushx storybook:baseline:chromatic)"
```
Comment (BeniCheni) [inline common/config/rush/command-line.json:95]: @craigpalermo, not familiar with rush. But what's this change after your suggested `rush change`?

Comment (craigpalermo) [reply common/config/rush/command-line.json:95]: I think that's a change @sdalonzo made, I think it should go away if you rebase on `master`

**Review by craigpalermo — APPROVED** [review]

**Pull request merged**